tests/test_geometry_load/test_geometry_load.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import os
+import sys
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+import openmoc
+from testing_harness import TestHarness
+from input_set import SimpleLatticeInput
+
+
+class SimpleLatticeTestHarness(TestHarness):
+    """An eigenvalue calculation in a 3D lattice with 7-group C5G7 data."""
+
+    def __init__(self):
+        super(SimpleLatticeTestHarness, self).__init__()
+        self.input_set = SimpleLatticeInput()
```

Comment:
A complex region tree would be like this:
zcylinder_s = openmoc.ZCylinder(x=0.0, y=0.0, radius=0.5, name='r_0.5')
zcylinder = openmoc.ZCylinder(x=0.0, y=0.0, radius=1.0, name='pin')
zcylinder1 = openmoc.ZCylinder(x=1.0, y=0.0, radius=0.5, name='pin1')

xmin = openmoc.XPlane(x=-2.0, name='xmin')
ymin = openmoc.YPlane(y=-2.0, name='ymin')
zmin = openmoc.ZPlane(z=-2.0, name='zmin')
xmax = openmoc.XPlane(x=2.0, name='xmax')
ymax = openmoc.YPlane(y=2.0, name='ymax')
zmax = openmoc.ZPlane(z=2.0, name='zmax')


fuel_s = openmoc.Cell(name='fuel_s')
fuel_s.setFill(materials['MOX-4.3%'])

h1 = openmoc.Halfspace(+1, zmin)
h2 = openmoc.Halfspace(-1, zmax)
h3 = openmoc.Halfspace(-1, zcylinder_s)
region0 = openmoc.Intersection()
region0.addNode(h1)
region0.addNode(h2)
region = openmoc.Intersection()
region.addNode(h3)
region.addNode(region0)
fuel_s.setRegion(region)


I1 = openmoc.Intersection()
U1 = openmoc.Union()
fuel = openmoc.Cell(name='fuel')
fuel.setFill(materials['UO2'])
h4 = openmoc.Halfspace(+1, zcylinder_s)
h5 = openmoc.Halfspace(-1, zcylinder)
h6 = openmoc.Halfspace(-1, zcylinder1)
I1.addNode(h4)
I1.addNode(h5)
I1.addNode(h1)
I1.addNode(h2)
U1.addNode(I1)
U1.addNode(h6)
fuel.setRegion(U1)